package.json

```diff
@@ -37,8 +37,8 @@
   },
   "dependencies": {
     "@adobe/adobe-client-data-layer": "^3.0.1",
-    "@adobe/magento-storefront-event-collector": "^1.8.0",
-    "@adobe/magento-storefront-events-sdk": "^1.8.0",
+    "@adobe/magento-storefront-event-collector": "^1.15.0",
+    "@adobe/magento-storefront-events-sdk": "^1.15.0",
```

Comment:
@SarikaVanapalli I noticed the SDK got bumped but I only see the event collector script was updated. Did you remember to run `npm install` once more before pushing?